rest_framework/generics.py

```diff
@@ -290,3 +291,20 @@ def patch(self, request, *args, **kwargs):
 
     def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
+
+
+def paginated_response(request, queryset, serializer_class, pagination_class=None, **kwargs):
+    if pagination_class is None:
+        pagination_class = api_settings.DEFAULT_PAGINATION_CLASS
+
+    paginator = pagination_class()
+    for attr, value in kwargs.items():
+        setattr(paginator, attr, value)
```

Comment:
That is really a hack that is pretty awesome :-)

I wanted to change the default lets say `page_size` without creating a new pagination class. So yes the proper way would be to create a new subclass. But if you are not ever reusing that pagination class, it is a lot of coding overhead.

I had one view where the page size was very small because the query was very big. So I wanted to change the page size to something that would not be used anywhere else.

I consider this pretty much inline with something like this:
https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_overrides

The proper way to override a widget would be to make a new form with a field attribute with the new widget. However, this is cumbersome because then you have to produce a form class and a field. So Django gives you a quick shortcut to override widgets.

Keep the simple things simple and the hard things doable. Updating page_size for one instance should be really easy.

With all that said, this is a pretty permissive hack that lets you stick anything on the pagination instance. If you wanted to reign it in some, I could see wanting to add a check for existing properties. I don't think a novice would know you could do this though, so leaving it open lets more experienced people do some cool stuff.